plugins/verifier/sample/sample.go

```diff
@@ -37,7 +38,14 @@ type PluginInputConfig struct {
 }
 
 func main() {
+	// send info and debug messages to stderr
+	fmt.Fprintln(os.Stderr, "info: initialized plugin")
+	fmt.Fprintf(os.Stderr, "debug: plugin %s %s \n", "sample", "1.0.0")
```

Comment:
suggestion: It would be nice if we had a plugin logger that could be used here instead of having to know the exact syntax to append to the front of the logs `debug: ` or `info: ` for the parsing to work

Reply:
Otherwise, there's more burden on plugin authors and we will need to have some good documentation about this

Reply:
Great idea, that makes a lot of sense. In your opinion, where's a good place to put this package? Or should it just be added to the skel package?

Reply:
yeah good question. My opinion would be to stick it in pkg/common/plugin